frontend/src/features/public-form/components/FormPaymentPage/stripe/components/DownloadReceiptBlock.tsx

```diff
@@ -82,36 +123,64 @@ export const DownloadReceiptBlock = ({
     window.location.href = getPaymentInvoiceDownloadUrl(formId, paymentId)
   }
   return (
-    <Box>
-      <Stack spacing="2rem">
+    <>
+      <Box bgColor="white" p="2rem">
         <Stack tabIndex={-1} spacing="0.75rem">
-          <Text textStyle="h2" color="content.strong">
-            Your payment has been made successfully.
+          <Text textStyle="h2" color="secondary.500">
+            Thank you, your payment has been made successfully.
           </Text>
-          <Text textStyle="subhead-1" color="content.strong">
-            Your form has been submitted and payment has been made.
+          <Text textStyle="subhead-1" color="secondary.500">
+            This is a message that the admins can customise on their thank you
```

Comment:
Is this message final? So many uses of `This is a message that ...` 🤣

Reply:
ahh I think this was an oversight from me - this should be editable by the form admin and should be tied to the issue of allowing the payment thank you page page to be editable (FRM-1560).  I'll put a placeholder text "Your form has been submitted and payment has been made." for now, and will look at how to retrieve the text when working on FRM-1560. 

thank you for catching this too 🙏🏼